content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Server Software
+description: A detailed guide on which server software is right for you
+---
+
+import { Card, Cards } from "fumadocs-ui/components/card";
+
+# Introduction
+
+Everybody has different requirements when setting up their server. This guide will help you decide which server software is right for you.
+Each server software has its own uses and features, so it's important to choose the right one for your needs.
+
+<Callout type="info" title="Note">
+  Server Software(s) are commonly referred to as "Server Jar(s)". This is
```

Comment:
> Server Software(s) are commonly referred to as "Server Jar(s)".

Server Software is commonly referred to as a "Server Jar(s)".

Explanation on "softwares" found further down, but in short, "software" is uncountable, so is not pluralized with an s.